piapage/__init__.py

```
@@ -430,8 +423,7 @@ def remote_medium_url_for_id(id, is_movie):
     def local_page_url_for_id(id):
```

Ruff (0.14.14)

Missing return type annotation for staticmethod `local_page_url_for_id`

(ANN205)